com.github.inercia.k3x.json

```diff
@@ -0,0 +1,120 @@
+{
+  "app-id": "com.github.inercia.k3x",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.36",
+  "sdk": "org.gnome.Sdk",
+  "command": "k3x",
+  "finish-args": [
+    "--device=dri",
+    "--share=ipc",
+    "--socket=x11",
+    "--socket=wayland",
+    "--socket=ssh-auth",
+    "--share=network",
+    "--env=LD_LIBRARY_PATH=/app/lib:/app/lib/x86_64-linux-gnu:/lib64:/lib:/usr/lib64:/usr/lib:/usr/local/lib",
+    "--filesystem=/run/docker.sock",
+    "--talk-name=org.freedesktop.Notifications",
+    "--talk-name=org.kde.StatusNotifierWatcher",
+    "--talk-name=com.canonical.indicator.application",
+    "--talk-name=org.ayatana.indicator.application",
+    "--filesystem=home:rw"
```

Comment:
Does it need full RW access or are there specific directories it uses?

Reply:
It could probably be fine tunned, but users can set a [_hook script_ that will be executed when the cluster is created/destroyed](https://github.com/inercia/k3x/blob/master/docs/user-manual-preferences.md#scripts). So that means k3x should be able to open and run those files. In addition, it needs to access:

- `~/.kube` for writing/readfing `kubeconfig` files
- `/.config/autostart` for adding k3x to the _autostart_ list